source/testnet/references/id-accounts.rst

```diff
@@ -1,16 +1,12 @@
 .. _Obtaining an identity: #obtaining-an-identity
 .. _Creating an account: #creating-an-account
 .. _Benefits: #benefits
-.. _Revoking anonymity: #revoking-anonymity
-.. _GTU: /testnet/docs/glossary#global-transaction-unit-gtu-
-.. _off-chain: /testnet/docs/glossary#off-chain
-.. _wallet: /testnet/docs/quickstart-wallet#before-you-start
-.. _Concordium ID: /testnet/docs/downloads#concordium-id
+.. _`anonymity revocation`: /testnet/references/id-accounts.html#revoking-anonymity
+.. _Concordium ID: /testnet/see-also/downloads.html#concordium-id
+.. _`initial account`: /testnet/see-also/glossary.html#initial-account
 .. _Notabene: https://notabene.id/
 .. _Onfido: https://onfido.com/
-.. _mainnet: /testnet/docs/glossary#mainnet
-.. _on-chain: /testnet/docs/glossary#on-chain
-.. _anonymity revocation: #revoking-anonymity
+.. _`mobile wallet`: /test/guides/get-started.html
```

Comment:
This link does not lead anywhere as far as I can see.

Moreover I don't think  we should use the term "mobile wallet". We use `Concordium ID`

Reply:
The link target is added in https://github.com/Concordium/concordium.github.io/pull/56. 

I'll change it to Concordium ID.